modules/virt-setting-cpu-allocation-ratio.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-assigning-compute-resources.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-setting-cpu-allocation-ratio_{context}"]
+= Setting the CPU allocation ratio
+
+The CPU Allocation Ratio specifies the degree of overcommitment by mapping vCPUs to time-slices of physical CPUs.
+
+For example, a mapping or ratio of 10:1 maps 10 virtual CPUs to 1 physical CPU using time-slices.
+
+To change the default number of vCPUs mapped to each physical CPU, set the `cpuAllocationRatio` value in the `HyperConverged` CR. The pod CPU request is calculated by multiplying the number of vCPUs by the reciprocal of the CPU allocation ratio. For example, if `cpuAllocationRatio` is set to 10, {VirtProductName} will request 10 times fewer CPUs on the pod for that VM.
+
+.Procedure
+
+Set the `cpuAllocationRatio` value in the `HyperConverged` CR to define a node CPU allocation ratio.
+
+. Open the `HyperConverged` CR in your default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Set the `cpuAllocationRatio`:
+
++
+[source,yaml]
+----
+...
+   spec:
+     configuration:
+       developerConfiguration:
+         cpuAllocationRatio: 10  <1>
```

Comment:
Current example is for KubeVirt, but in HCO we use another parameter: `vmiCPUAllocationRatio`

And the yaml spec should be:

```
spec:
  resourceRequirements:
    vmiCPUAllocationRatio: 10
```